README.md

```diff
@@ -8,45 +8,51 @@
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/okta)
 ![Code Style](https://img.shields.io/badge/Code%20Style-flake8-informational.svg)
 
-
 # Okta Python Management SDK
-* [Release status](#release-status)
-* [Need help?](#need-help)
-* [Getting started](#getting-started)
-* [Usage guide](#usage-guide)
-* [Configuration reference](#configuration-reference)
-* [Building the SDK](#building-the-sdk)
-* [Contributing](#contributing)
+
+- [Release Status](#release-status)
+- [Need help?](#need-help)
+- [Getting Started](#getting-started)
+- [Usage Guide](#usage-guide)
+- [Pagination](#pagination)
+- [Configuration Reference](#configuration-reference)
+- [Rate Limiting](#rate-limiting)
+- [Building the SDK](#building-the-sdk)
+- [Contributing](#contributing)
 
 This repository contains the Okta management SDK for Python. This SDK can be used in your server-side code to interact with the Okta management API and
-* Create and update users with the [Users API](https://developer.okta.com/docs/api/resources/users)
-* Add security factors to users with the [Factors API](https://developer.okta.com/docs/api/resources/factors)
-* Manage groups with the [Groups API](https://developer.okta.com/docs/api/resources/groups)
-* Manage applications with the [Apps API](https://developer.okta.com/docs/api/resources/apps)
-* Much more!
+
+- Create and update users with the [Users API][users-api-docs]
+- Add security factors to users with the [Factors API][factors-api-docs]
+- Manage groups with the [Groups API][groups-api-docs]
+- Manage applications with the [Apps API][apps-api-docs]
+- Much more!
 
 > Requires Python version 3.6.0 or higher.
 
 You can also learn more on the [Okta + Python][lang-landing-page] page in our documentation.
 
 ## Release status
-This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/).
 
-| Version | Status                             |
-| ------- | ---------------------------------- |
-| 0.x     |  :warning: Beta Release (Retired)  |
-| 1.x     |  :heavy_check_mark: Release        |
-| 2.x     |  :construction: Pre-Release        |
+This library uses semantic versioning and follows Okta's [Library Version Policy][okta-library-versioning].
+
+| Version | Status                           |
+| ------- | -------------------------------- |
+| 0.x     | :warning: Beta Release (Retired) |
+| 1.x     | :heavy_check_mark: Release       |
+| 2.x     | :construction: Pre-Release       |
```

Comment:
Is this table right?  I don't think you are in pre-release for 2.0, but rather release for 1.x?